project.clj

```diff
@@ -8,18 +8,8 @@
 
   :dependency-sets [:clojure :common :clanhr]
 
-  :dependencies [[commons-logging "1.2"]
-                 [clj-bugsnag "0.2.3"]
-                 [clj-librato "0.0.5"]
-                 [analytics-clj "0.3.0"]
-
-                 ;; disable analytics-clj log
-                 [org.slf4j/slf4j-nop "1.7.12"]
-
-                 ;; turn on for major analytics logs
-                 ;[org.slf4j/jcl-over-slf4j "1.7.5"]
-                 ;[ch.qos.logback/logback-classic "1.0.13"]
-                 ;kkk[ch.qos.logback/logback-core "1.0.13"]
-                 ]
+  :dependencies [[clj-bugsnag "0.2.9"]
+                 [clanhr/clj-librato "0.0.5"]
+                 [analytics-clj "0.3.0"]]
```

Comment:
Yes.. I thought about it, but they are specific to this module, and not shared by any other module. So I don't know...